Sources/Testing/Events/Clock.swift

```diff
@@ -338,3 +338,74 @@ extension Test.Clock.Instant {
 #endif
   }
 }
+
+extension Test.Clock.Instant: Codable {
+
+  /// The keys used to encode a ``Test.Clock.Instant```.`
+  private enum _CodingKeys: CodingKey {
+    /// Encodes and decodes to ``uptime`` on Darwin, ``suspending`` on other
+    /// platforms.
+    case suspending
+
+    /// Encodes and decodes the wall clock time.
+    case wall
+  }
+
+  /// A ``Codable`` container for a ``timespec``. This is a helper type to
```

Comment:
Nitpick, but important for DocC: Add two newlines after the first sentence, otherwise the entire paragraph appears as the brief for the type:

```
/// A ``Codable`` container for a ``timespec``.
///
/// This is a helper type to [...]
```